Code/Editor/MainWindow.cpp

```diff
@@ -2137,6 +2160,291 @@ bool MainWindow::focusNextPrevChild(bool next)
     return QMainWindow::focusNextPrevChild(next);
 }
 
+void MainWindow::InitializeActionContext()
+{
+    ActionContextProperties contextProperties;
+    contextProperties.m_name = "O3DE Editor";
+
+    m_actionManagerInterface->RegisterActionContext("", EditorMainWindowActionContextIdentifier, contextProperties, this);
+}
+
+void MainWindow::InitializeActions()
+{
+    auto cryEdit = CCryEditApp::instance();
+
+    // --- Level Actions
+
+    // New Level
+    {
+        ActionProperties actionProperties;
+        actionProperties.m_name = "New Level";
+        actionProperties.m_description = "Create a new level";
+        actionProperties.m_category = "Level";
```

Comment:
Yes that is actually one of the things I was hoping we'd support sooner rather than later to streamline action registration, but it doesn't look like we do right now.